src/Salt/SiteBundle/Entity/CommentRepository.php

```diff
@@ -76,4 +84,42 @@ public function findByTypeItem(array $id): array
     {
         return $this->findByItem($id['itemType'].':'.$id['itemId']);
     }
+    
+     /**
+     * @param string $itemType
+     * @param int $itemId
+     *
+     * @return Comment
+     */    
+    
+    public function exportComments($itemType,$itemId)
+    {
+        $query= $this->getEntityManager()
+                     ->createQueryBuilder();
+        if($itemType=='item')
+        {
+            $query->select('r,u,o,i');
+        }
+        else 
+        {
+           $query->select('r,u,o,d');
+        }
+        $query->from('SaltSiteBundle:Comment', 'r') 
+              ->innerJoin('r.user','u')
+              ->innerJoin('u.org','o');
```

Comment:
Is there a reason to add this join? why would you need to do this? you only need the item id

Reply:
In story acceptance criteria mentioned organization name column too. And organization name can get only by joining tables.

Reply:
As I said in the previous comment, the Comment entity has a relationship with the User and the User has a relationship with the Organization, so you should be able to get the Organization object from the User object ($user->getOrg();`)